tests/.env-dist

```diff
@@ -4,6 +4,7 @@
 #COMPOSE_FILE=docker-compose.yml:docker-compose.caching.yml
 #COMPOSE_FILE=docker-compose.yml:docker-compose.mysql.yml
 #COMPOSE_FILE=docker-compose.yml:docker-compose.pgsql.yml
+#COMPOSE_FILE=docker-compose.yml:docker-compose.mysql.yml:docker-compose.pgsql.yml:docker-compose.caching.yml
```

Comment:
Should we uncomment this line by default, so if a developer copies this file to `.env` he get's a full featured test environment?

Reply:
Yes, I think so.